src/orderbook.js

```diff
@@ -1197,6 +1334,37 @@ OrderBook.prototype.computeAutobridgedOffers = function() {
   this._offersAutobridged = autobridgeCalculator.calculate();
 };
 
+OrderBook.prototype.computeAutobridgedOffersWrapper = function() {
+  var startTime = Date.now();
+  this.computeAutobridgedOffers();
+  this.mergeDirectAndAutobridgedBooks();
+  var lasted = (Date.now() - startTime) * 5;
+  let m = this._autobridgeThrottleTimeMultiplier;
+  while (lasted > m * OrderBook.AUTOBRIDGE_CALCULATE_THROTTLE_TIME) {
+    ++m;
```

Comment:
Don't we also need a mechanism for allowing this to decrease back to the original?